cli/cli/kurtosis_config/overrides_objects/v6/grafana_loki_config_v6.go

```diff
@@ -0,0 +1,10 @@
+package v6
+
```

Comment:
Might be worth adding the header, "DO NOT CHANGE THIS FILE!" that the other files have